src/ui/components/shared/LoadingScreen.tsx

```diff
@@ -8,18 +9,29 @@ import { LoadingTips } from "./LoadingTips";
 import ReplayLogo from "./ReplayLogo";
 import { BubbleViewportWrapper } from "./Viewport";
 
+const colorOptions: Array<"blue" | "green" | "red"> = ["blue", "green", "red"];
+
 export function LoadingScreenTemplate({
   children,
-  showTips,
+  showTips = true,
 }: {
   children?: ReactNode;
   showTips?: boolean;
 }) {
+  const [hoverboardColor, setHoverboardColor] = useState(colorOptions[2]);
+
+  const handleHoverboardClick = () => {
+    const randomIndex = Math.floor(Math.random() * colorOptions.length);
+    setHoverboardColor(colorOptions[randomIndex]);
+  };
+
   return (
     <BubbleViewportWrapper>
       <div className="relative flex w-96 flex-col items-center space-y-1 rounded-lg bg-loadingBoxes p-8 py-12 shadow-sm">
         <div className="flex flex-col items-center space-y-8">
-          <ReplayLogo size="md" />
+          <div className="w-32" onClick={handleHoverboardClick}>
+            <Hoverboard color={hoverboardColor} />
```

Comment:
I love this, but wonder if we want to play it safe and use a dynamic import. @markerikson can you help share an example?

Reply:
Hm looks like we're seeing a deploy error